src/Pages/ListActivities.php

```diff
@@ -26,93 +29,96 @@ abstract class ListActivities extends Page implements HasForms
 
     protected static Collection $fieldLabelMap;
 
-    public function mount($record)
+    #[Url]
+    public int $perPage = 10;
+
+    public function mount($record): void
     {
         $this->record = $this->resolveRecord($record);
     }
 
-    public function getBreadcrumb(): string
+    public function getBreadcrumb(): string|Htmlable
     {
         return static::$breadcrumb ?? __('filament-activity-log::activities.breadcrumb');
     }
 
-    public function getTitle(): string
+    public function getTitle(): string|Htmlable
     {
         return __('filament-activity-log::activities.title', ['record' => $this->getRecordTitle()]);
     }
 
     public function getActivities()
     {
         return $this->paginateTableQuery(
-            $this->record->activities()->with('causer')->latest()->getQuery()
+            $this->getActivitiesQuery()
         );
     }
 
+    protected function getActivitiesQuery(): Builder
+    {
+        return $this->record->activities()
+            ->with(['causer' => fn($query) => $query->withTrashed()])
```

Comment:
Makes sense, but this will throw an error if the causer doesn't implement `SoftDeletes`, so we need to check that first.